packages/lib-classifier/src/components/Classifier/components/MetaTools/components/FavouritesButton/FavouritesButton.js

```diff
@@ -0,0 +1,49 @@
+import counterpart from 'counterpart'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import theme from 'styled-theming'
+import { Icon } from 'grommet-icons'
+import zooTheme  from '@zooniverse/grommet-theme'
+import { PlainButton } from '@zooniverse/react-components'
+import HeartIcon from './HeartIcon'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+export const Favourite = styled(HeartIcon)`
+  fill: ${props => props.filled ? '#E45950' : 'none'};
```

Comment:
Is there a convention for defining colours in the theme, rather than using hex codes like this?

Reply:
There is: https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-grommet-theme/src/index.js#L28

Named colours here can be referenced as strings in Grommet props

Reply:
Oh wait but that's for grommet - you could import the grommet theme object in this component and get it from there

Reply:
Cool. Looking around at examples, `theme.colors.icons` seems to be a convention so I'll look at that for the Zoo Theme.

Reply:
We could define our own icon sizes in the theme too. Grommet's `small` looks slightly bigger, to my eye, than Becky's design.